css-three-column-layout/styles.css

```diff
@@ -0,0 +1,56 @@
+* {
+  background-color: rgba(25, 150, 218, 255);
+  box-sizing: border-box;
+  font-family: "lato", sans-serif;
+  color: #fff;
+}
+
+.container {
+  max-width: 1000px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.column-third {
+  float: left;
+  width: 33.33%;
+  color: aliceblue;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.image-container {
+  width: 128px;
+  height: 128px;
+  margin: 20px 96px 20px 96px;
+  position: relative;
+  float: left;
+  border-radius: 75%;
+  overflow: hidden;
+  background: #fff;
+}
+
+img {
+  padding: 20px;
+  background-color: white;
+  margin-left: 12%;
+  margin-top: 12%;
```

Comment:
This is a lot of margin. I'm sure this is to center the image in its container, but you can accomplish this by making the `div` container a flexbox, and applying `justify-content` and `align-items`, each with a value of `center`.

Reply:
I'm not sure I understand what you meant here, but I think it is important that I understand, so please tell me if I implemented this incorrectly.

I set up a css selector for 'column-thirds' (line 14) and I set up a selector for .row-three-column (row 20). Now this selector contains the property, display, which allows it become a 'flex-container', which since those columns are the respective parents of the image containers, the image containers become 'flex-boxes'. Now since those '.image-container' (line 25) are 'flex-boxes' or 'flexitems' the properties of 'align-items' and 'justify-content' work on them. This seems to me to be important so at the risk of being repetitive - I'm saying this for myself but also hoping you'll confirm - because the image containers parent is set to 'display: flex' and is thereby a flex container, the image-container is a flex item and align-items and justify content are valid properties to work with in it. _Therefore, the image can be centered within the image container._ is that correct?

If this is not what you were looking for please let me know.

Reply:
![image](https://user-images.githubusercontent.com/51938797/111238742-ed7e0a80-85b4-11eb-8ac7-7e5de7ea98df.png)
![image](https://user-images.githubusercontent.com/51938797/111238752-f53daf00-85b4-11eb-8512-d86ec933e007.png)